src/main/java/org/jenkinsci/plugins/mesos/JenkinsScheduler.java

```diff
@@ -407,6 +418,10 @@ private void createMesosTask(Offer offer, Request request) {
     LOGGER.info("Launching task " + taskId.getValue() + " with URI " +
                 joinPaths(jenkinsMaster, SLAVE_JAR_URI_SUFFIX));
 
+    if (isExistingTask(offer, taskId)) {
```

Comment:
Do you know, when is this possible to get into this state? i.e., a createMesosTask() is called with an existing task?